plugins/org.jkiss.dbeaver.registry/src/org/jkiss/dbeaver/registry/policy/ApplicationPolicyService.java

```diff
@@ -67,4 +67,11 @@ private void disableCommand(@NotNull ICommandService commandService, @NotNull St
         }
     }
 
+    /**
+     * Return true, if software install/update policy enabled
+     */
+    public boolean isPolicySoftwareInstallUpdateEnabled() {
```

Comment:
Naming hell.
- POLICY_SOFTWARE_INSTALL actually means "install disabled"
- isPolicySoftwareInstallUpdateEnabled actually means "install update disabled"

Reply:
Get it, fixed to name isInstallUpdateDisabled()